### ISSUE TITLE
rule sort-keys add suggestions

### ISSUE THREAD

**Issue by aladdin-add** [eslint/eslint#14021]
it has been brought up a few times, but not accepted as it's likely being unsafe.
now, eslint introduced [suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions), so it seems good to have.

related issue: https://github.com/eslint/eslint/issues/7714 https://github.com/eslint/eslint/issues/11542

**What rule do you want to change?**
sort-keys
**Does this change cause the rule to produce more or fewer warnings?**
same, but add suggestions
**How will the change be implemented? (New option, new default behavior, etc.)?**
n
**Please provide some example code that this change will affect:**

```js
var obj = {c: '', b: '', a: ''}
```
suggest to
```
var obj = {a: '', b: '', c: ''}
```

**What does the rule currently do for this code?**
Expected object keys to be in ascending order. 'a' should be before 'b'
**What will the rule do after it's changed?**
same error, but providing suggestions.
**Are you willing to submit a pull request to implement this change?**
y

**Comment by nzakas** [comment]
Looks like we lost track of this, sorry! We are only doing bug fixes for stylistic rules at this point, otherwise the rules are considered frozen, so closing.
